lego/apps/events/tasks.py

```diff
@@ -184,11 +190,7 @@ def registration_payment_save(self, result, registration_id, logger_context=None
         registration.charge_amount = result["amount"]
         registration.charge_status = result["status"]
         registration.save()
-        notify_user_payment(
-            constants.SOCKET_PAYMENT_SUCCESS,
-            registration,
-            success_message="Betaling gjennomført",
-        )
+        return {"client_secret": result["client_secret"]}
```

Comment:
Is this safe to pass around with celery?
https://stripe.com/docs/api/payment_intents/object#payment_intent_object-client_secret
> The client secret can be used to complete a payment from your frontend. _It should not be stored_, ...

Reply:
I think it should be ok, it's not logged anywhere, and the only thing you can do with it is pretty much confirm a payment.